README.md

```diff
@@ -369,6 +388,6 @@ An auto post dependency is one computed from what you actually used, rather than
 
 A cached command is where the inputs/outputs for a command are tracked, and the command is treated as a pure function and skipped if its inputs didn't change. This feature is more useful in build systems that go forward (from inputs to outputs) rather than the standard build systems that go from outputs to inputs.
 
-### Regenerate [Make]
+### Regenerate [Make, Gup]
```

Comment:
Oh. I misunderstood the paragraph; it doesn't.

On Tue, Feb 16, 2016 at 9:48 PM, Neil Mitchell <notifications@github.com>
wrote:

> In README.md
> <https://github.com/ndmitchell/build-shootout/pull/24#discussion_r53075773>
> :
>
> > @@ -369,6 +388,6 @@ An auto post dependency is one computed from what you actually used, rather than
> >
> >  A cached command is where the inputs/outputs for a command are tracked, and the command is treated as a pure function and skipped if its inputs didn't change. This feature is more useful in build systems that go forward (from inputs to outputs) rather than the standard build systems that go from outputs to inputs.
> >
> > -### Regenerate [Make]
> > +### Regenerate [Make, Gup]
>
> Does gup really let you replace a running gup file and as a result
> restarts from the beginning?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ndmitchell/build-shootout/pull/24/files#r53075773>.
>



-- 
Noé Rubinstein.